.github/workflows/ci.yaml

```diff
@@ -80,9 +80,71 @@ jobs:
         name: coverage
         path: "${{ env.COVERAGE_FILE }}"
 
+  pathogen-ci:
+    runs-on: ubuntu-latest
+    continue-on-error: true
+    env:
+      repodata_use_zst: true
+    strategy:
+      max-parallel: 4
+      matrix:
+        include:
+          - { pathogen: avian-flu, build-args: auspice/flu_avian_h5n1_ha.json }
+          - { pathogen: ebola }
+          - { pathogen: lassa }
+          - { pathogen: monkeypox }
+          - { pathogen: mumps }
+          - {
+              pathogen: ncov,
+              build-args: all_regions -j 2 --profile nextstrain_profiles/nextstrain-ci,
+            }
+          - {
+              pathogen: seasonal-flu,
+              build-args: --configfile profiles/ci/builds.yaml -p,
+            }
+          - { pathogen: zika }
+    name: test-pathogen-repo-ci (${{ matrix.pathogen }})
+    defaults:
+      run:
+        shell: bash -l {0}
+    steps:
+      - uses: actions/checkout@v3
+      - uses: mamba-org/provision-with-micromamba@v15
+        with:
+          environment-file: false
+          environment-name: augur
+          extra-specs: nextstrain-base
+          channels: nextstrain,conda-forge,bioconda
+          cache-env: true
+      - run: pip install .
+      - run: mamba list
+
+      - uses: actions/checkout@v3
+        with:
+          repository: nextstrain/${{ matrix.pathogen }}
```

Comment:
These two checkouts are overlapping. I doubt that's the desired behaviour, and it's likely to cause issues at some point.

Reply:
Addressed in f1a17bce7c4473be7d03a5d824f499813b98b15d.